examples/coreos/units-etcd/ceph-rgw@.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Rados Gateway Service
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+Environment=RGW_NAME=%H
+ExecStartPre=-/usr/bin/docker kill %p
+ExecStartPre=-/usr/bin/docker rm %p
+ExecStart=/usr/bin/docker run --rm --name %p -p 80:80 \
+   --net=host \
```

Comment:
rgw works without it but etcd needs for now, because of this: https://github.com/ceph/ceph-docker/blob/master/daemon/config.kv.sh#L121. If localhost is used, kviator won't be able to connect as there is no etcd inside container. And it will go to dead loop on line 121. 

Without --net=host, people need to specify the real IP of the host, which is not feasible, or etcd server IP in KV_IP, instead of 127.0.0.1. We probably want to avoid hard code etcd server IP in the unit file. 

Do you think it will be appropriate to update get_config function to accommodate this use case. If KV_TYPE is enabled and KV_IP is 127.0.0.1, then get docker host IP instead of using 127.0.0.1. The docker host IP can be retrieved by `/sbin/ip route|awk '/default/ { print $3 }'`.